docs/integrations-in-rancher/cluster-api/overview.md

```diff
@@ -0,0 +1,210 @@
+- `rancher-turtles-system/rancher-turtles-controller-manager`
+- `rancher-turtles-system/rancher-turtles-cluster-api-operator`
+- `capi-system/capi-controller-manager`
+
+### Via Helm Install
+
+This section walks through different Helm installation options for Rancher Turtles depending on CAPI operator use:
+- [Install Rancher Turtles with Cluster API Operator as a dependency](#install-rancher-turtles-with-cluster-api-operator-as-a-helm-dependency).
+- [Install Rancher Turtles without Cluster API Operator](#install-rancher-turtles-without-cluster-api-operator-as-a-helm-dependency).
+
+The Cluster API Operator is required for installing Rancher Turtles and you can choose whether you want to take care of this dependency yourself or let the Rancher Turtles Helm chart manage it for you. We recommend [installing as a dependency](#install-rancher-turtles-with-cluster-api-operator-as-a-helm-dependency) for the sake of simplicity, but the best option may depend on your specific configuration.
+
+CAPI Operator allows for handling the lifecycle of Cluster API providers using a declarative approach, extending the capabilities of `clusterctl`. If you want to learn more about it, you can refer to [Cluster API Operator book](https://cluster-api-operator.sigs.k8s.io/).
+
+:::info prerequisite
+Before installing Rancher Turtles in your Rancher environment, Rancher's `embedded-cluster-api` functionality must be disabled. This includes also cleaning up Rancher-specific webhooks that otherwise would conflict with CAPI ones.
+
+To simplify setting up Rancher for installing Rancher Turtles, the official Rancher Turtles Helm chart includes a `pre-install` hook that applies these changes which are outlined below:
+- Disable the `embedded-cluster-api` feature in Rancher.
+- Delete the `mutating-webhook-configuration` and `validating-webhook-configuration` webhooks that are no longer needed.
+:::
+
+#### Install Rancher Turtles with `Cluster API Operator` as a Helm dependency
+
+The `rancher-turtles` chart is available in https://rancher.github.io/turtles and this Helm repository must be added before proceeding with the installation:
+
+```bash
+helm repo add turtles https://rancher.github.io/turtles
+helm repo update
+```
+
+As mentioned before, installing Rancher Turtles requires the [Cluster API Operator](https://github.com/kubernetes-sigs/cluster-api-operator) and the Helm chart can handle its installation automatically with a minimum set of flags:
+
+```bash
+helm install rancher-turtles turtles/rancher-turtles --version v0.5.0 \
+    -n rancher-turtles-system \
+    --dependency-update \
+    --create-namespace --wait \
+    --timeout 180s
+```
+
+This operation could take a few minutes and once completed you can review the installed controllers listed below:
+- `rancher-turtles-controller`
+- `capi-operator`
+
+:::note
+- If `cert-manager` is already available in the cluster, you can disable its installation as a Rancher Turtles dependency to avoid conflicts:
```

Comment:
```suggestion
- If `cert-manager` is already available in the cluster, to avoid conflicts, disable its installation as a Rancher Turtles dependency:
```